### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by jukent**
Take home content for Part 1 of Python Zero to Thirty Beginner Tutorial -- First Python Script.

We are considering rolling out parts as they are ready. Please take a look and add any additional explanations or examples you think are necessary.

### CODE REVIEW

File: curricula/self_paced/part_1/z230-1c.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+1. [python] Okay, now that you've read the data in a way that
+   is easy to modify later, it is time to actually do something 
+   with the data.
+   
+   Compute the *wind chill* factor, which is the cooleing 
+   effect of the wind. As wind speed increases the rate at 
+   which a body loses heat increases. The formula for this is:
+   
+   WCI = a + (b * Temperature) - (c * WindSpeed^2) + 
+      (d * Temperature * WindSpeed^2)
```
Comment (kmpaul) [inline curricula/self_paced/part_1/z230-1c.md]: I'm not sure how to render this better.  Not sure what the best approach is...  However, if markdown can't render equations nicely, then I think we might want to move this content over to Jupyter Notebooks.  They can display Latex equations.

File: curricula/self_paced/part_1/z230-1c.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+   Think of GitHub as online storage for versions of your 
+   project, much like hosting your code in a Google Drive, 
+   but with better features specific to coding. A lot of 
+   GitHub's features show their usefulness when you are 
+   working collaboratively, sharing your code with other 
+   scientists, or if you wanted to display and easily 
+   visualize changes in your code between commits. 
+   
+==========================
```
Comment (kmpaul) [inline curricula/self_paced/part_1/z230-1c.md]: Again.  Not sure how best to render this, but maybe this won't be markdown in the end, anyway.

File: curricula/self_paced/requirements.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+Requirements & Installation
+--------------------------
+
+First, check that you have conda or miniconda installed 
+on your OS. 
+
+1. [bash] Check your conda version:
+
+   ```bash
+   $ conda list anaconda
```
Comment (kmpaul) [inline curricula/self_paced/requirements.md]: This should be:

```suggestion
   $ conda --version
```

File: curricula/self_paced/requirements.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+   ```
+   At the time of writing this, the latest version of conda is 
+   4.6. If you have an old version of conda installed, update
```
Comment (kmpaul) [inline curricula/self_paced/requirements.md]:
```suggestion
   4.8. If you have an old version of conda installed, update 
```

File: curricula/self_paced/requirements.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+2. [bash] If necessary, update:
+
+   ```bash
+   $ conda update
+   ```
+
+   If you don't have conda installed at all, please install it. 
+
+   Use the instructions at 
+   https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html
```
Comment (kmpaul) [inline curricula/self_paced/requirements.md]: Probably should go at the top.

**Review by kmpaul — APPROVED** [review]
I suggested some changes, but I'm thinking that we can implement *most* of those changes later.  I think that we might want to re-write these as Jupyter Notebooks, so we get all of the advantages of them.  Maybe we don't have any code cells, but the advantages of Notebooks over markdown might make it worth it.

File: curricula/self_paced/part_1/z230-1c.md (hunk 1 above)
Comment (jukent) [reply curricula/self_paced/part_1/z230-1c.md]: That's a good idea.

Comment (jukent) [reply curricula/self_paced/part_1/z230-1c.md]: This is done in the .ipynb files

**Pull request merged**